docassemble/mlhframework/data/questions/mlh_interview_framework.yml

```diff
@@ -91,11 +91,14 @@ default screen parts:
 
   #pre: This is the pre screen part.
   #submit: This is the submit screen part.
-  post: Provided by [Michigan Legal Help](https://michiganlegalhelp.org).
-  #under: This is the under screen part.
+  # post: Provided by [Michigan Legal Help](https://michiganlegalhelp.org).
   #right: This is the right screen part.
   global footer: This is the footer part.
   main page footer: This is the main footer.
+  under: |
+    % if not user_logged_in():
+    [:key: Sign in](${url_of('login', next=interview_url())}) or [sign up](${url_of('register', next=interview_url())}) to return later
```

Comment:
I do like shorter text, but also feel like it's useful to include something like "save your answers" or "save your progress" per the existing text. 

If kept as-is, add period at the end.

Reply:
That's fair, I can change that part back. Even though it's optional, I'll still probably remove "optional" to shorten it up a bit.